tests/ga/test_extension.py

```diff
@@ -3302,9 +3304,13 @@ def manifest_location_handler(url, **kwargs):
         with mock_wire_protocol(self.test_data, http_get_handler=manifest_location_handler) as protocol:
             ext_handlers = protocol.get_goal_state().extensions_goal_state.extensions
 
-            with self.assertRaises(ExtensionDownloadError):
-                protocol.client.fetch_manifest(ext_handlers[0].manifest_uris,
-                    timeout_in_minutes=0, timeout_in_ms=200)
```

Comment:
The timeout_in_minutes and timeout_in_ms parameters were used by test code only, so I removed them. Now the test code uses _DOWNLOAD_TIMEOUT instead.